contracts/contracts/system_contracts/kip247/GaslessSwapRouter.sol

```diff
@@ -10,6 +10,15 @@ import "@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol";
 import "./IKIP247.sol";
 import "./IWKAIA.sol";
 
+/**
+ * @title GaslessSwapRouter
+ * @dev Implements KIP-247 gasless transaction functionality
+ * This contract allows users to swap ERC20 tokens for KAIA without possessing any KAIA.
```

Comment:
This contract per se does not allow swap without possessing KAIA.
```suggestion
 * This contract allows users to swap ERC20 tokens for KAIA for gasless transaction.
```

Reply:
fixed